target/linux/ramips/mt7620/target.mk

```diff
@@ -4,7 +4,7 @@
 
 SUBTARGET:=mt7620
 BOARDNAME:=MT7620 based boards
-FEATURES+=usb ramdisk
+FEATURES+=usb ramdisk small_flash
```

Comment:
I'm sorry but it's not my favorite... This will affect the whole target. Besides, this might work for now, but the kernel size will keep growing gradually as the version goes up, so it won't be a fundamental solution. I wish there would be a workaround by changing U-Boot configuration or something like that.

Reply:
No argument here, but no luck getting U-Boot changed (so far at least). There was a two step procedure noted as possible - but not sure that should be gating (as it is a bit longer term issue, like you note). But feel free to disagree.

Reply:
BTW, one other thought on this - perhaps dumb, but ... :-). Why not make this feature device dependent, not an entire target ("family")? Different HW has different features, but currently this is limited to a whole range of products at once. Just a thought!